service/openapi.yaml

```diff
@@ -201,6 +201,8 @@ components:
       required:
         - flags
       properties:
+        metadata:
+          $ref: "#/components/schemas/metadata"
```

Comment:
Please move this after the flags property.

Reply:
Done